site.mk

```diff
@@ -168,12 +167,15 @@ GLUON_TLWDR4900_SITE_PACKAGES := $(USB_PACKAGES_BASIC) $(TOOLS_PACKAGES) $(USB_P
 
 DEFAULT_GLUON_RELEASE := snapshot~$(shell date '+%Y%m%d')
 
+# default to 11s meshing
+GLUON_WLAN_MESH = 11s
```

Comment:
not sure if this syntax is correct... also, if you want to use the default, you could simply leave this out completely.